### PR TITLE
[ros_google_cloud_language] Fix for python3; Avoid attribute error

### PR DESCRIPTION
**Pull request by tkmtnt7000**
**Upstream dependendy**: https://github.com/ros/rosdistro/pull/38053

I fixed analyze_text.py for python3.
- Avoid attribute error `AttributeError: 'bytes' object has no attribute 'encode'`
- Convert map function to list comprehensions for python3
- Use unidecode to convet non-ascii character to ascii character

I would like to add a test code to confirm whether change is valid or not, but I think it's difficult to add test on Github action because this node needs google credential json.

### CODE REVIEW

File: ros_google_cloud_language/node_scripts/analyze_text.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 import actionlib
 import rospy
 import os
+import sys
+from unidecode import unidecode
```
Comment (mqcmd196) [inline ros_google_cloud_language/node_scripts/analyze_text.py:12]: This import failes in `python2`, so please add `try`, `except`

**Review by mqcmd196 — CHANGES_REQUESTED** [review]

File: ros_google_cloud_language/node_scripts/analyze_text.py (hunk 1 above)
Comment (tkmtnt7000) [reply ros_google_cloud_language/node_scripts/analyze_text.py]: Thank you. Updated. [cc249d2](https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/488/commits/cc249d29e611fc563119efbab6a1905e5a2307e8)

**Comment by tkmtnt7000** [comment]
Upstream PR has already merged.

**Review by mqcmd196 — APPROVED** [review]
lgtm

**Pull request merged**